src/testUtils.js

```diff
@@ -47,25 +66,10 @@ function runTest(dirName, transformName, options, testFilePrefix) {
   );
   // Assumes transform is one level up from __tests__ directory
   const module = require(path.join(dirName, '..', transformName + '.js'));
-  // Handle ES6 modules using default export for the transform
-  const transform = module.default ? module.default : module;
-
-  // Jest resets the module registry after each test, so we need to always get
-  // a fresh copy of jscodeshift on every test run.
-  let jscodeshift = require('./core');
-  if (module.parser) {
-    jscodeshift = jscodeshift.withParser(module.parser);
-  }
-
-  const output = transform(
-    {path: inputPath, source},
```

Comment:
The one thing that feels a bit gross in this change is that inline tests aren't able to have a path here. It seems like most transforms only use source, but people will be unable to use inline tests with their transform if they use the file path directly.